### PR TITLE
add support for separated global config

### PR DESCRIPTION
**Pull request by otakumike**
implements #47

### CODE REVIEW

File: conf/struct.go [hunk 1]
```diff
@@ -8,6 +8,9 @@ package conf
 // This is to ensure they both conform to the golang convention
 // and that they are able to be properly parsed by the `tok config-x` commands
 type Config struct {
+	Global struct {
+		Syncservice string `yaml:"syncservice"`
```
Comment (yjimk) [inline conf/struct.go]: This isn't a bool? What's the value expected for this field?

File: initialize/main.go [hunk 2]
```diff
@@ -81,3 +110,18 @@ func createDotTok() {
 		}
 	}
 }
+
+func createGlobalDotTok() {
+	h, err := homedir.Dir()
+	if err != nil {
+		log.Fatalln("Unable to resolve home directory, unable to initialise. Sorry!")
```
Comment (yjimk) [inline initialize/main.go]: Optional, but i"d cut the "Sorry!"

File: initialize/main.go [hunk 3]
```diff
@@ -5,23 +5,64 @@ import (
...
+}
+
+func emojiDefaults() bool {
+	return true
+}
+
+func readGlobalConfig() {
+	h, err := homedir.Dir()
+	if err != nil {
+		log.Fatalln("Unable to resolve home directory so can't initialise Tokaido. Sorry!")
```
Comment (yjimk) [inline initialize/main.go]: I'd cut the "Sorry!" from here

File: conf/struct.go (hunk 1 above)
Comment (otakumike) [reply conf/struct.go]: It will be "docker" or "fusion"

Comment (otakumike) [reply conf/struct.go]: and maybe later we'll add others (eg: nfs)

**Pull request merged**